### PR TITLE
Ks 2021 06 add login screen

### PR DESCRIPTION
**Pull request by Rineee**
There is now a login screen, but once you logged in with a valid account on aplus dev, it should store the token an no more login is required..

### CODE REVIEW

File: containers/Auth/StartUpScreen.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export const StartUpScreen = (props) => {
+
+  useEffect(() => {
+    const tryLogin = async () => {
+      const authToken = await AsyncStorage.getItem('authToken');
+      if (!authToken) {
+        props.navigation.navigate('Auth');
+      }
+      else {
```
Comment (khamui) [inline containers/Auth/StartUpScreen.js:16]: just a little thing, but we should align on the same brace style for if-statements.
I also like the stroustrup brace style. @Rineee @phillimorland @fuzzylogic2000 if you don't mind using this style i will add a check to eslint?

**Review by khamui — COMMENTED** [review]
Very cool! Logging in works ⚡️
just some small things, i commented below.

**Comment by khamui** [comment]
Actually my comments are not relevant to the functionality. It works, so if you want to postpone these little changes to the next sprint let me know so i merge immediately @Rineee .

File: containers/Auth/StartUpScreen.js (hunk 1 above)
Comment (fuzzylogic2000) [reply containers/Auth/StartUpScreen.js:16]: So this is the stroustrup brace style? Everything sound complicated for js. :rofl: Yes, I like it (looks a bit like python :stuck_out_tongue:), let's use it!

File: navigation/IdeaNavigator.js [hunk 2]
```diff
@@ -13,4 +15,18 @@ const IdeaNavigator = createStackNavigator({
...
+const AuthNavigator = createStackNavigator({
+  Authentication: LoginScreen
+});
+
+const StartUpNavigator = createStackNavigator({
+  StartUp: StartUpScreen
+});
+
+const MainNavigator = createSwitchNavigator({
+  StartUp: StartUpNavigator,
```
[the remark this answers is not in the document]
Comment (Rineee) [reply navigation/IdeaNavigator.js:27]: Yes, you are right! It somehow didnt work when I first tried it, thats why I added those StackNavigators around, but I just tried again and it works without, dont know what I was doing :see_no_evil:

**Review by fuzzylogic2000 — APPROVED** [review]
Whoop whoop!

File: containers/Auth/StartUpScreen.js (hunk 1 above)
Comment (khamui) [reply containers/Auth/StartUpScreen.js:16]: haha 😝

**Pull request merged**